### PR TITLE
Internationalize folder names

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra/pull/1217
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/400
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/267
https://github.com/open-orchestra/open-orchestra-migration-bundle/pull/20
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/774

### CODE REVIEW

File: MigrationBundle/Migrations/Version20170307181737.php [hunk 1]
```diff
@@ -267,6 +268,34 @@ protected function createFolderPath()
...
+     *
+     * @param Database $db
+     */
+    protected function updateFolderNames(Database $db)
+    {
+        $this->write('  + Updating folderNames');
+
+        $this->checkExecute($db->execute('
+            var backLanguages = ["'
+                . implode('", "', $this->container->getParameter('open_orchestra_base.administration_languages'))
```
Comment (alavieille) [inline MigrationBundle/Migrations/Version20170307181737.php]: use json_encode

**Review by alavieille — COMMENTED** [review]
else comment +1

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**